server/package.json

```diff
@@ -97,6 +97,7 @@
     "node-schedule": "^1.3.2",
     "nodemailer": "^6.6.0",
     "nodemailer-mailgun-transport": "^2.1.3",
+    "p-retry": "4",
```

Comment:
is the latest version 5 problematic or have other dependency issues?

https://www.npmjs.com/package/p-retry?activeTab=versions

Reply:
We'd need to switch to ESM in node. From the author: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Seems like something we should do but I still need to look into it more

Reply:
Gotcha, thanks! Transitioning to ESM seems like something worth doing at some point but no need to hold this up.

Reply:
I've done some looking into it and it still seems like a bit of a dumpster fire that I wouldn't want to touch just yet. The previous versions of these packages that have gone pure ESM seem stable enough to still make up 98% of npm downloads so we have some time